src/modals/MarkModal.tsx

```diff
@@ -137,38 +93,99 @@ const MarkModal = (props: PropsMarkModal) => {
                 backdropDismiss={false}
                 isOpen={props.isOpen}
             >
-                <IonContent color="light">
-                    <IonCardContent class="align-center">
-                        <IonList class="transparent">
-                            <DatePicker date={date} onChange={setDate} color={"basic"} title={"Start of last period"} />
-                            <IonItem lines="full" class="transparent">
-                                <IonLabel color="basic">Period length</IonLabel>
-                                <IonSelect
-                                    class="mark"
-                                    interfaceOptions={selectOptions}
-                                    placeholder=""
-                                    onIonChange={(ev) => {
-                                        setPeriod(Number(ev.detail.value.id));
-                                    }}
-                                >
-                                    {period_days.map((day) => (
-                                        <IonSelectOption key={day.id} value={day}>
-                                            {day.name}
-                                        </IonSelectOption>
-                                    ))}
-                                </IonSelect>
-                            </IonItem>
-                            <Buttons
-                                period={period}
-                                setPeriod={setPeriod}
-                                date={date}
-                                setDate={setDate}
-                                closeModal={closeModal}
-                            />
-                        </IonList>
-                    </IonCardContent>
-                </IonContent>
-            </IonModal>
+                <IonDatetime
+                    ref={datetimeRef}
+                    color="dark-basic"
+                    presentation="date"
+                    locale="en-GB"
+                    size="cover"
+                    preferWheel={false}
```

Comment:
Here and below you don't need to write `false` or `true` explicitly. You can just write `preferWheel` without `={false}`

Reply:
Ok!